src/plots/cartesian/graph_interact.js

```diff
@@ -313,9 +313,15 @@ function hover(gd, evt, subplot) {
                 .map(function(pi) { return pi.id; })),
 
         xaArray = subplots.map(function(spId) {
+            if(spId.substr(0,7) === 'ternary') {
```

Comment:
good idea. Actually, since it explicitly depends on a `_ternary` attribute I'll just look for that.